test/externalTests/fishing.js

```diff
@@ -23,7 +23,7 @@ module.exports = () => (bot, done) => {
     })
   })
 
-  bot.on('playerCollect', (collector, collected) => {
+  bot.once('playerCollect', (collector, collected) => {
```

Comment:
This is not the correct way of checking this in my opinion. If `collected.type` is not 'object' then the test would fail due to timeout. 

My guess is that you catch two items, hence `playerCollect` called twice. I don't know if it's possible so there may be an option that it needs to be inspected more closely. In such case it deserves it's own issue in my opinion.

Reply:
Right, It could be the Ink sac tho the probability is 1.2%, and It happened in both my tests which is suprising
> **1.11**: Ink sac chances have been changed from 10 chances for 1 ink sac to 1 chance for 10 ink sacs.
https://minecraft.gamepedia.com/Fishing

Reply:
I haven't looked closer, but the check of `collected.type` is there for some reason. If you catch it simply once it may still fail and the `collected.type` is not as expected then a timeout will occur and the test fails once again.